wg-opcodes/summaries/current-status.md

```diff
@@ -0,0 +1,56 @@
+# Metadata
+
+Title:            Current Status of OpCodes Workgroup
+Workgroup:        OpCodes
+Document Version: 1.0
+
+# Document Purpose
+
+This document is intended to provide a single source of truth of the status of ongoing op codes
+discussions relative to the upcoming hardfork. Any discussions not related to the latest upcoming
+fork will be referenced and postponed until the next hardfork or removed entirely if the discussion
+is in a complete form and no spec is planned to be proposed.  Implementation details or specs of
+particular op codes is out of scope of this document.
```

Comment:
I may reword this last sentence to something like:
Implementation details or specs of particular op codes is out of scope of this document, however they will be referenced.